newdle/client/src/components/answer/Calendar.js

```diff
@@ -144,12 +147,24 @@ Hours.defaultProps = {
   hourStep: 2,
 };
 
-export default function Calendar({minHour, maxHour}) {
+export default function Calendar({defaultMinHour, defaultMaxHour}) {
+  const [[minHour, maxHour], setHourSpan] = useState([defaultMinHour, defaultMaxHour]);
   const answers = useSelector(getAnswers);
   const timeSlots = useSelector(getNewdleTimeslots);
   const duration = useSelector(getNewdleDuration);
   const activeDate = toMoment(useSelector(getActiveDate), HTML5_FMT.DATE);
   const dispatch = useDispatch();
+  const defaultHourSpan = defaultMaxHour - defaultMinHour;
+
+  useEffect(() => {
```

Comment:
I am wondering whether we need to use `useEffect` here. Its dependencies will probably never change.